src/coreclr/vm/eventtrace.cpp

```diff
@@ -4680,6 +4680,12 @@ extern "C"
             {
                 ETW::EnumerationLog::EnumerateForCaptureState();
             }
+
+            if (g_fEEStarted && !g_fEEShutDown)
```

Comment:
It occurs to me that this should go into EtwCallbackCommon so that it runs when an EventPipe (e.g. dotnet-trace) session is triggered as well.

Reply:
Fixed